Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/TransformComponent.cpp

```diff
@@ -954,6 +955,15 @@ namespace AzToolsFramework
                 return AZ::Failure(AZStd::string("You cannot set an entity to be a child of a closed container!"));
             }
 
+            // Don't allow entities to be parented outside their container.
+            if (m_focusModeInterface)
+            {
+                if (!m_focusModeInterface->IsInFocusSubTree(actualValue))
+                {
+                    return AZ::Failure(AZStd::string("You can only set a parent as one of the entities belonging to the focused prefab!"));
```

Comment:
nit: remove the '!' at the end to keep the message tone neutral.

Reply:
Removed exclamation marks from all messages in function.